### PR TITLE
Document dep_cats_permits

### PR DESCRIPTION
**Pull request by mgraber**

### CODE REVIEW

File: recipes/dep_cats_permits/create.sql [hunk 1]
```diff
@@ -1,3 +1,26 @@
+/*
+DESCRIPTION:
+    1. Import geocoded dep_cats_permits to EDM database using PSTDIN
+    2. Filter based on ID and request type
```
Comment (mgraber) [inline recipes/dep_cats_permits/create.sql:4]: Does this logic make sense?

File: recipes/dep_cats_permits/create.sql [hunk 2]
```diff
@@ -1,3 +1,26 @@
...
+        - exclude cancelled
+        - exclude G permits
+        - exclude C permits unless they are non-registrations
+        - exclude CA permits unless they are not work permits and not expired
```
Comment (mgraber) [inline recipes/dep_cats_permits/create.sql:8]: Does this logic make sense?

Comment (SPTKL) [reply recipes/dep_cats_permits/create.sql:8]: this logic was written based on a email from EARD, unfortunately it's not documented anywhere else

Comment (mgraber) [reply recipes/dep_cats_permits/create.sql:8]: @AmandaDoyle I know EARD requested C permit research files. Do we know whether we are excluding all C-permits? Does the logic look correct to you?

**Review by SPTKL — APPROVED** [review]

File: recipes/dep_cats_permits/create.sql (hunk 2 above)
Comment (AmandaDoyle) [reply recipes/dep_cats_permits/create.sql:8]: I'd compare the outputs to make sure that it returns the same records.  Looking at past issues (https://github.com/NYCPlanning/ceqr-app-data-archive/issues/79 and https://github.com/NYCPlanning/ceqr-app-data-archive/issues/80) 
There should be an output for research **Air: DEP data subset for research #80**
All C-permits should be excluded from final output.  We can reconfirm with EARD

**Pull request merged**